swarm-derive/src/lib.rs

```diff
@@ -50,8 +50,8 @@ fn build_struct(ast: &DeriveInput, data_struct: &DataStruct) -> TokenStream {
     let net_behv_event_proc = quote! {::libp2p::swarm::NetworkBehaviourEventProcess};
     let either_ident = quote! {::libp2p::core::either::EitherOutput};
     let network_behaviour_action = quote! {::libp2p::swarm::NetworkBehaviourAction};
-    let into_protocols_handler = quote! {::libp2p::swarm::IntoConnectionHandler};
-    let protocols_handler = quote! {::libp2p::swarm::ConnectionHandler};
+    let into_connections_handler = quote! {::libp2p::swarm::IntoConnectionHandler};
```

Comment:
```suggestion
    let into_connection_handler = quote! {::libp2p::swarm::IntoConnectionHandler};
```
Also for the other renamed variables: it was called Protocol**s**Handler, but renamed to ConnectionHandler (without the **s**, since it only handles a single connection, not multiple). Sorry for nitpicking.

Reply:
No problem @elenaf9 I appreciate the thorough review! I have made the changes.